pynucastro/rates/library.py

```diff
@@ -397,31 +397,43 @@ def backward(self):
         return only_bwd
 
     def derived_forward(self):
+        """
```

Comment:
can you make the docstring more descriptive?  What is the purpose of this function now?

Reply:
The idea of derived_forward is to exclude the reaclib reverse, tabular and weak rates. Which are quite pathological at this stage.

Reply:
but the name implies that the are then the list of forward rates that have already been rederived from the reverse rates -- is that the intent?

Reply:
well, there is a point of confusion here, but is not easy to solve:

1. We want to classify forward reactions as exothermic (Q>0), and reverse by endothermic (Q<0). However, we ReacLib doesn't follow this path. If a reaction is measured experimentally (independent of the Q) then they use detailed balance to get the opposite direction.

2. Eventually I want to classify forward and reverse by positive Q and negative Q; however, for testing purposes, making this classification may eventually lead to computing the detailed balance twice. 

3. The idea of derived_forward is to eliminate the reverse and weak, and see if our job gives the same Reaclib predictions. check NSE convergence with the pf functions. In the future, I will move this function to a unit test.

Reply:
okay, can you make the docstrings in the code clear as to what they intend to do now?  and we can address the confusion in a latter PR